staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3603,6 +3603,37 @@ func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, load
 	return sgList, setupSg, nil
 }
 
+// sortELBSecurityGroupList returns a list of sorted securityGroupIDs based on the original order
+// from buildELBSecurityGroupList. The logic is:
+//  * securityGroups specified by ServiceAnnotationLoadBalancerSecurityGroups appears first in order
```

Comment:
Can you clarify that the auto-created or `CloudConfig.Global.ElbSecurityGroup` SG comes in the middle?